packages/connect/src/http_client.ts

```diff
@@ -101,59 +97,93 @@ export class HttpClient implements Client {
     /**
      * Retrieve a specific order from the API
      * @param   orderHash     An orderHash generated from the desired order
-     * @return  The SignedOrder that matches the supplied orderHash
+     * @return  The APIOrder that matches the supplied orderHash
      */
-    public async getOrderAsync(orderHash: string): Promise<SignedOrder> {
+    public async getOrderAsync(orderHash: string, requestOpts?: RequestOpts): Promise<APIOrder> {
+        if (!_.isUndefined(requestOpts)) {
+            assert.doesConformToSchema('requestOpts', requestOpts, clientSchemas.requestOptsSchema);
+        }
         assert.doesConformToSchema('orderHash', orderHash, schemas.orderHashSchema);
-        const responseJson = await this._requestAsync(`/order/${orderHash}`, HttpRequestType.Get);
-        const order = relayerResponseJsonParsers.parseOrderJson(responseJson);
+        const httpRequestOpts = {
+            params: requestOpts,
+        };
+        const responseJson = await this._requestAsync(`/order/${orderHash}`, HttpRequestType.Get, httpRequestOpts);
+        const order = relayerResponseJsonParsers.parseAPIOrderJson(responseJson);
         return order;
     }
     /**
      * Retrieve an orderbook from the API
      * @param   request         An OrderbookRequest instance describing the specific orderbook to retrieve
-     * @param   requestOpts     Options specifying page information, defaults to { page: 1, perPage: 100 }
+     * @param   requestOpts     Options specifying page information, and network id.
```

Comment:
Are there any defaults?

Reply:
Yes, but the defaults should be specified at the relayer level. I can re-add them here for extra consistency but yea the way the API is defined is that they have defaults if they are omitted, whereas this client was explicitly setting them to their default value.